.github/workflows/check_bootnodes_enrs.yml

```diff
@@ -0,0 +1,69 @@
+name: Check Bootnodes ENRs
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check_enrs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install protobuf-compiler
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+
+      - name: Install enr-cli
+        run: cargo install enr-cli
+
+      - name: Check bootnodes ENRs for padding
+        run: |
+          files_with_padding=""
+          for FILE in $(find . -type f \( -name 'bootnodes.yaml' -o -name 'bootstrap_nodes.txt' \)); do
+            echo "Checking paddings in file: $FILE"
+            while read -r LINE; do
+              if [[ "$LINE" =~ enr: ]]; then
+                BASE64_STRING=${LINE#enr:}
+                if [[ "$BASE64_STRING" =~ = ]]; then
+                  if [[ "$files_with_padding" != *"$FILE"* ]]; then
+                    echo "Error: ENR in $FILE contains padding: $LINE"
+                    files_with_padding="$files_with_padding $FILE"
+                  fi
+                fi
+              fi
+            done < "$FILE"
+          done
+          if [ -n "$files_with_padding" ]; then
+            echo "Error: Bootnodes ENR strings contain padding in the following files:"
+            echo "$files_with_padding"
+            exit 1
+          fi
+
+      - name: Check ENR Strings with enr-cli
+        run: |
+          for FILE in $(find . -type f \( -name 'bootnodes.yaml' -o -name 'bootstrap_nodes.txt' \)); do
+            echo "Testing decoding in file: $FILE"
+            while read -r LINE; do
+              if [[ "$LINE" =~ enr: ]]; then
+                ENR_LINE=$(echo "$LINE" | sed -e 's/#.*//' -e 's/^[ \t]*//' -e 's/[ \t]*$//')
+                if [[ "$ENR_LINE" =~ ^- ]]; then
+                  BASE64_STRING=$(echo "$ENR_LINE" | awk -F "enr:" '{print $2}' | awk '{gsub(/"/, "", $0); print}')
+                else
+                  BASE64_STRING=${ENR_LINE#enr:}
+                  BASE64_STRING=$(echo "$BASE64_STRING" | awk '{gsub(/"/, "", $0); print}')
+                fi
```

Comment:
This check should be dependent on the file type, right? Checking if the line starts with a dash only works if we're actually using dashes in `yaml` and not using them in `txt`.

I think this should read all the lines that are not comments or empty and then check for the proper format. Maybe even use a `yaml` decoder to make sure that the file has the correct format?

Reply:
Yes, condition satisfied only for `yaml` files. 
You mean include case when strings in `yaml` file will be not dash-prefixed to not skip them if we will change formatting ?